pkgs/development/python-modules/pki-core/default.nix

```
@@ -0,0 +1,50 @@
+{ stdenv, fetchurl, buildPythonPackage, sphinx, rpmextract, nss, requests,
+  six }:
+
+buildPythonPackage rec {
+  name = "pki-core-${version}";
```

```suggestion
  pname = "pki-core";
```
This makes things much nicer for users pinning this to a different version in an overlay, and matches the general coding standards.